projects/packages/publicize/src/class-publicize-utils.php

```diff
@@ -0,0 +1,83 @@
+<?php
+/**
+ * Publicize_Utils.
+ *
+ * @package automattic/jetpack-publicize
+ */
+
+namespace Automattic\Jetpack\Publicize;
+
+use Automattic\Jetpack\Connection\Manager;
+use Automattic\Jetpack\Modules;
+
+/**
+ * Publicize_Utils class.
+ */
+class Publicize_Utils {
+
+	/**
+	 * Whether the current page is the social settings page.
+	 */
+	public static function is_social_settings_page() {
+		$screen = get_current_screen();
+
+		return ! empty( $screen ) && 'jetpack_page_jetpack-social' !== $screen->base;
+	}
+
+	/**
+	 * Whether the current page is the Jetpack settings page.
+	 */
+	public static function is_jetpack_settings_page() {
+		$screen = get_current_screen();
+
+		return ! empty( $screen ) && 'toplevel_page_jetpack' !== $screen->base;
+	}
+
+	/**
+	 * Whether the block editor should have the social features.
+	 *
+	 * @return bool
+	 */
+	public static function should_block_editor_have_social() {
+		if ( ! is_admin() ) {
+			return false;
+		}
+
+		$screen = get_current_screen();
+
+		if ( empty( $screen ) || ! $screen->is_block_editor() ) {
+			return false;
+		}
+
+		if ( ! self::is_connected() || ! self::is_publicize_active() ) {
```

Comment:
Does this mean if the Jetpack connection doesn't exist for one admin, but the other admin is successfully connected Publicize won't be active?

Reply:
No, [`has_user_connected`](https://github.com/Automattic/jetpack/blob/de290e975184f20b3dfbf602e40a713d4c2785b2/projects/packages/connection/src/class-manager.php#L624-L626) means if any user connection exists, which means if any of the admins has a user connection set up, publicize will be visible.